manifest-build-tools/application/reprove.py

```diff
@@ -243,15 +426,38 @@ def main():
 
     if args.git_credential:
         manifest_actions.set_git_credentials(args.git_credential)
-
+    
```

Comment:
optional parameters also can be set inside init()
create a function set_args or just add code in init
```
__init__(self, args)
.....
...
set_args(args)
...
```